introduction.md

```diff
@@ -0,0 +1 @@
+Hello I am happy to get started with git and to try out this tutorial
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.